arangodb-net-standard/ViewApi/Models/SearchAliasIndex.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ArangoDBNetStandard.ViewApi.Models
+{
+    public class SearchAliasIndex
+    {
+        /// <summary>
+        /// Possible value for <see cref="Operation"/>
+        /// </summary>
+        public const string SearchAliasIndexAddOperation = "add";
+
+        /// <summary>
+        /// Possible value for <see cref="Operation"/>
+        /// </summary>
+        public const string SearchAliasIndexDelOperation = "del";
+
+        /// <summary>
+        /// Possible value for <see cref="Operation"/>
+        /// </summary>
+        public const string SearchAliasIndexNoOperation = "";
+
+        /// <summary>
+        /// Name of the collection
+        /// </summary>
+        public string Collection { get; set; }
+
+        /// <summary>
+        /// Name of the index
+        /// </summary>
+        public string Index { get; set; }
+
+        /// <summary>
+        /// Optional. Type of operation.
+        /// Possible values:
+        /// <see cref="SearchAliasIndexAddOperation"/>,
+        /// <see cref="SearchAliasIndexDelOperation"/>, or
+        /// <see cref="SearchAliasIndexNoOperation"/>
+        /// </summary>
+        public string Operation { get; set; }
```

Comment:
`Operation` is apparently only relevant when updating the properties of a View definition. Should we mention it?

https://github.com/arangodb/docs/blob/main/3.10/release-notes-api-changes310.md#search-alias-views
> Options for partially changing properties (PATCH /_api/view/<view>/properties), to add or remove inverted indexes from the View definition:

Reply:
Done.